android/src/main/java/com/klippa/reactscanner/KlippaScannerSDKModule.kt

```diff
@@ -310,8 +305,81 @@ class KlippaScannerSDKModule(
                 builder.imageAttributes.storeImagesToGallery = config.getBoolean("StoreImagesToCameraRoll")
             }
 
-            val klippaScanner = builder.build(reactContext)
-            currentActivity.startActivity(klippaScanner)
+            val modes = ArrayList<KlippaDocumentMode>()
+
+            if (config.hasKey("CameraModeSingle")) {
+                val cameraModeSingle = config.getMap("CameraModeSingle") ?: return
+
+                var name = "Single Document"
```

Comment:
This is unnecessary because `KlippaSingleDocumentMode` `name` and `Instructions` `message` have default values, so if no value is provided have the SDK handle it instead.